content/agent/proxy.md

```diff
@@ -29,22 +29,66 @@ A few options are available to send traffic to Datadog over SSL/TLS for hosts th
 
 Traditional web proxies are supported natively by the Agent. If you need to connect to the Internet through a proxy, edit your Agent configuration file.
 
-##### Agent v6
+### Agent v6
+
+With Agent v6 you can set different proxy servers for `https` and `http` requests in your `datadog.yaml` file. The Agent always uses `https` to send data to Datadog, but integrations might use `http` to collect metrics from different software.
+No matter the proxied queries, you can activate SSL or not on your proxy server. Find below some configuration examples for your `datadog.yaml` file:
+
+Setting an HTTP proxy for all `https` queries:
+
+```
+proxy:
+    https: http://<SSL_PROXY_SERVER_FOR_HTTPS>:<PORT>
+```
 
-Edit the `datadog.yaml` file with your proxy information. Use the `no_proxy` list to specify hosts that should bypass the proxy.
+Setting an HTTPS proxy for both `https` and `http` queries:
 
 ```
 proxy:
-    http: http://user:password@proxy_for_http:port
-    https: http://user:password@proxy_for_https:port
-#   no_proxy:
-#     - host1
-#     - host2
+    https: https://<SSL_PROXY_SERVER_FOR_HTTPS>:<PORT>
+    http: https://<SSL_PROXY_SERVER_FOR_HTTP>:<PORT>
 ```
 
+Setting a `<USERNAME>` and `<PASSWORD>` to contact the proxy server for both `https` and `http` queries:
+
+```
+proxy:
+    https: http://<USERNAME>:<PASSWORD>@<PROXY_SERVER_FOR_HTTPS>:<PORT>
+    http: http://<USERNAME>:<PASSWORD>@<PROXY_SERVER_FOR_HTTPS>:<PORT>
+```
+
+Using the `no_proxy` list to specify hosts that must bypass the proxy:
+
+```
+proxy:
+    https: http://<USERNAME>:<PASSWORD>@<PROXY_SERVER_FOR_HTTPS>:<PORT>
+    http: http://<USERNAME>:<PASSWORD>@<PROXY_SERVER_FOR_HTTPS>:<PORT>
+    no_proxy:
+      - host1
+      - host2
+```
+
+#### Proxy with environment variables
+
+Starting with Agent v6.4, you can set you proxy settings through environment variables:
+
+- `DD_PROXY_HTTPS`: Sets a proxy server for `https` queries.
+- `DD_PROXY_HTTP`: Sets a proxy server for `http` queries.
+- `DD_NO_PROXY`: Sets a list of host that should bypass the proxy (a space-separated list).
+
+Environment variables have precedence over values in the `datadog.yaml` file. If the environment variables are present with an empty value (e.g. ``DD_HTTP_PROXY=""``), the Agent uses those empty value instead of ignoring it and using lower-precedence options.
```

Comment:
Environment variables have precedence over values in the `datadog.yaml` file. If the environment variables are present with an empty value (e.g. ``DD_HTTP_PROXY=""``), the Agent uses those empty values instead of lower-precedence options.